fields/default/fields_list.json

```diff
@@ -857,6 +859,7 @@
             "string"
         ],
         "fillable_by": [
+            "ramiosago.base.ext"
         ]
```

Comment:
Hmmm... Just noticied that fields `insurance.osago.items[].date.start` and `insurance.osago.items[].date.end` in `fields_list.json` wasn't fillable by any source(before my changes). But in json schema of report we can see that this fields are fillable by `ramiosago.base` source. And this source really provides data for this field. I think this is error, that should be fixed.

Reply:
Approved. Need to add `ramiosago.base` source to `fillable_by` array